pontoon/base/templates/translate.html

```diff
@@ -109,6 +109,9 @@
 
             <li class="langpack"><a href=""><i class="fa fa-cloud-download fa-fw"></i>Download Language Pack</a></li>
             <li class="download"><i class="fa fa-cloud-download fa-fw"></i>Download Translations</i></li>
+            <li class="download-tmx" data-url-template="{{ url('pontoon.download_tmx', locale='locale', slug='project', filename='locale.project') }}">
+                <i class="fa fa-cloud-download fa-fw"></i>Download Translation Memory</i>
+            </li>
```

Comment:
You should move this block above `.download`, so that `.download` and `.upload` will ne next to each other.